xmake/modules/target/action/install/main.lua

```diff
@@ -123,7 +143,7 @@ end
 
 -- install shared libraries
 function _install_shared_libraries(target, opt)
-    local bindir = target:is_plat("windows", "mingw") and target:bindir() or target:libdir()
+    local bindir = _get_target_bindir(target, opt)
```

Comment:
Please do not mix libdir and bindir.

https://github.com/xmake-io/xmake/blob/9563841bfc304874cc9d308b1e29bd6cde76631a/xmake/plugins/pack/batchcmds.lua#L247